### PR TITLE
Pulling actual case values from JHUDataSource

### PR DESCRIPTION
**Pull request by ghop02**
This PR addresses issue https://github.com/covid-projections/covid-data-model/issues/110

Still need to clean this up a bit more, but basically swaps out reading the jhu files from the latest file to using the JHU data source as a common timeseries. 

As a result some of the column names changed and needed to add country level fips.

I still need to verify the output to make sure it matches with the existing/surface any differences. 

### Please Check
- [ ] Will the *current* schema in [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.md) be updated with this PR?
- [ ] Are tests passing?

### CODE REVIEW

File: libs/build_dod_dataset.py [hunk 1]
```diff
@@ -49,145 +52,82 @@ def _get_abbrev_df():
...
         .merge(interventions_df, left_on="state", right_on="state", how="inner")
     )
-
     counties_remapped = counties_decorated.rename(
         columns=OUTPUT_COLUMN_REMAP_TO_RESULT_DATA
     )
     counties = pd.DataFrame(counties_remapped, columns=RESULT_DATA_COLUMNS_COUNTIES)
     counties = counties.fillna(NULL_VALUE)
     counties.index.name = "OBJECTID"
+    counties["Province/State"] = counties["Province/State"].map(abbrev_us_state)
```
Comment (goldblatt) [inline libs/build_dod_dataset.py:90]: i mean would love to use the class vars here if we have them?

File: libs/build_dod_dataset.py [hunk 2]
```diff
@@ -49,145 +52,82 @@ def _get_abbrev_df():
...
-            "Active": "sum",
-            "Country/Region": "first",
-            "Latitude": "first",
-            "Longitude": "first"
-            # People tested is currently null
-            #'People Tested': 'sum'
-        }
-    )
-
-    # basically the states_agg has full state names, the interventions have abbreviation so we need these to be merged
```
Comment (goldblatt) [inline libs/build_dod_dataset.py:166]: why does this change?

Comment (ghop02) [reply libs/build_dod_dataset.py:166]: This is all handled in the Data source/timeseries preprocessing, the datasets coming out of it are already one row per state

File: libs/build_dod_dataset.py (hunk 1 above)
Comment (ghop02) [reply libs/build_dod_dataset.py:90]: word. I think I'll be able to use your latest PR for that right?

File: libs/build_dod_dataset.py [hunk 3]
```diff
@@ -49,145 +52,82 @@ def _get_abbrev_df():
...
+    return data.latest_values(AggregationLevel.STATE)
 
 
 def get_usa_by_county_with_projection_df(input_dir, intervention_type):
     us_only = _get_usa_by_county_df()
-    fips_df = FIPSPopulation.local().data  # used to get interventions
-    interventions_df = (
-        _get_interventions_df()
-    )  # used to say what state has what interventions
+    fips_df = FIPSPopulation.local().data
```
Comment (goldblatt) [inline libs/build_dod_dataset.py:68]: do we need to specify this is county data?

Comment (ghop02) [reply libs/build_dod_dataset.py:68]: it works okay because the left join is only starting with counties, but might be good to make it explicit.

**Pull request closed without merging**